src/main/java/com/contrastsecurity/http/UrlBuilder.java

```diff
@@ -99,7 +104,7 @@ public String getNotesByApplicationUrl(String organizationId, String appId, Stri
         return String.format("/ng/%s/applications/%s/traces/%s/notes?expand=skip_links", organizationId, appId, traceId, formString);
     }
 
-    public String getVulnTagsByApplicationUrl(String organizationId, String appId) throws UnsupportedEncodingException {
+    public String getVulnTagsByApplicationUrl(String organizationId, String appId) {
```

Comment:
I know it's a little strange, but the rest of this file uses "Trace" instead of "Vuln" or "Vulnerability". Can you refactor the method name to `getTraceTagsByApplicationUrl`